feed_resources/app/src/main/res/values/strings.xml

```diff
@@ -1,21 +1,25 @@
 <resources>
-    <string name="app_name">FW Discovery</string>
```

Comment:
This was old leftover from the times when we had separate app for every feed resoruce.